extensions/resource-deployment/src/ui/resourceTypeWizard.ts

```diff
@@ -59,12 +73,19 @@ export class ResourceTypeWizard {
 		} else if (instanceOfAzureSQLDBDeploymentProvider(this.provider)) {
 			return new DeployAzureSQLDBWizardModel(this.provider, this);
 		}
-		// other types are undefined for now.
-		return undefined;
+		return new PageLessDeploymentModel(this.provider, this);
 	}
 
 	public async open(): Promise<void> {
-		this.model.initialize();
+		this.wizardObject = azdata.window.createWizard(this.resourceType.displayName, this.resourceType.name, 'wide');
+		this.setPages([]);
```

Comment:
this problably belongs in an initialize method or if this object does not need or have an initialize method then in the contructor as well.